4/Dockerfile

```diff
@@ -19,10 +19,21 @@ WORKDIR /var/www/html
 RUN	apk add --no-cache libpq
```

Comment:
Please also get rid of this line when you make the `scanelf` adjustments. Thanks.